### PR TITLE
Add github issue form templates

### PR DESCRIPTION
**Pull request by NukeManDan**
This introduces [template forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms), with a very strong call to action to use our stack exchange over filing issues for support. This can beset be experienced before a merge at the docs issues:

https://github.com/substrate-developer-hub/substrate-docs/issues/new/choose

image

All buttons lead to open a issue that has this to begin with:

image

---

If users want to bypass using the issue forms and get the typical empty text entry area they only need to hit the bottom left "open a blank issue" link (a bit hidden)

image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    attributes:
+      label: Description of bug
+      # description: What seems to be the problem?
+      # placeholder: Describe the problem.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to Reproduce
```
Comment (sacha-l) [inline .github/ISSUE_TEMPLATE/bug.yaml:30]:
```suggestion
      label: Steps to reproduce
```

**Review by sacha-l — APPROVED** [review]
LGTM - good initiative. Minor edit for lower case label.

**Review by shawntabrizi — APPROVED** [review]
Can we make the stackexchange link the first option?

**Comment by NukeManDan** [comment]
Sadly I don't see a way to configure this. But SE is linked to in every issue template and you must click to confirm you have looked in all the issues here and SE before you enter text on an issue. So it's very blunt about it existing and what it's for I think.

**Comment by NukeManDan** [comment]
bot merge

**Pull request merged**